css/components/button.css

```diff
@@ -0,0 +1,89 @@
+/* === css for button component === */
+
+button{
```

Comment:
you can give an extra class of btn instead of using a button tag.

Reply:
alright.. will add that feature.